### PR TITLE
Create tejas-luthra.json

### PR DESCRIPTION
**Pull request by tejas-luthra**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

**Review by wdhdev — APPROVED** [review]

**Comment by tejas-luthra** [comment]
Am I missing something?

**Comment by WilliamDavidHarrison** [comment]
> Am I missing something?

Just wait for a maintainer to merge the pr

**Comment by tejas-luthra** [comment]
I checked the format of the file and I did not see any errors. Can someone tell me what I missed anything? Any help will be appreciated.

File: domains/tejas-luthra.json
Comment (WilliamDavidHarrison) [inline domains/tejas-luthra.json:6]:
```suggestion
        "email": "tejasluthra3008@gmail.com"
```

Found the issue

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by WilliamDavidHarrison** [comment]
> I checked the format of the file and I did not see any errors. Can someone tell me what I missed anything? Any help will be appreciated.

https://github.com/is-a-dev/register/pull/4158#pullrequestreview-1218537267

**Review by wdhdev — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**
